clients/go/go.mod

```diff
@@ -3,88 +3,142 @@ module github.com/oasisprotocol/sapphire-paratime/clients/go
 go 1.19
 
 require (
-	github.com/ethereum/go-ethereum v1.10.26
-	github.com/oasisprotocol/emerald-web3-gateway v1.3.2-0.20220819151005-ed83ad4ae5be
+	github.com/ethereum/go-ethereum v1.13.4
+	github.com/oasisprotocol/oasis-web3-gateway v1.3.2-0.20230807080150-da1635399a26
```

Comment:
We barely use this part.

Reply:
Where does the v1.3.2-0 come from?

Reply:
> If we copied that into the client would that cleanup the dependency graph much?

Yeah. It would.

Hmm. Let's go for it, especially considering that the response and error structs aren't defined by the gateway either.

Reply:
Yeah, I think this struct won't really change much, and the savings are probably worth it.